server/game/core/gameSteps/abilityWindow/TriggerWindowBase.ts

```diff
@@ -276,7 +276,7 @@ export abstract class TriggerWindowBase extends BaseStep {
     }
 
     private canAnyAbilitiesResolve(triggeredAbilities: TriggeredAbilityContext[]) {
-        return triggeredAbilities.some((triggeredAbilityContext) => triggeredAbilityContext.ability.hasAnyLegalEffects(triggeredAbilityContext, true));
+        return triggeredAbilities?.some((triggeredAbilityContext) => triggeredAbilityContext.ability.hasAnyLegalEffects(triggeredAbilityContext, true));
```

Comment:
Had to add this since `triggeredAbilities` was coming in a `undefined` for some of the tests. Its occurring when `let abilitiesToResolve = this.unresolved.get(this.currentlyResolvingPlayer);` in `server\game\core\gameSteps\abilityWindow\TriggerWindowBase.promptUnresolvedAbilities()` doesn't have any unresolved abilities for the currently resolving player. It didn't seem serious to me, but let me know if you disagree.